invenio_assets/filters.py

```diff
@@ -105,6 +108,25 @@ def input(self, _in, out, **kw):
         self.subprocess([self.binary or 'cleancss'], out, _in)
 
 
+class CleanCSS4Filter(CleanCSSFilter):
```

Comment:
we inherit from the CleanCSSFilter class and we just override what we need. so:
- the name
- the open method to use the option `--rebaseTo` instead of `--root`